<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBottomCtrlWidget</class>
 <widget class="QWidget" name="MainBottomCtrlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>45</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>45</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1162</width>
    <height>45</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnQuery">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Query</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrevious">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>46</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLock">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>46</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Lock</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrint">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLIS">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>LIS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCheck">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnCheck">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>UnCheck</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditInfo">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Edit Info.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditResult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Edit Res.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSwitch">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
